src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -182,72 +188,86 @@ class ACHContractStep extends React.Component {
                     guidesCallTaskID={CONST.GUIDES_CALL_TASK_IDS.WORKSPACE_BANK_ACCOUNT}
                     shouldShowBackButton
                 />
-                <ReimbursementAccountForm
-                    reimbursementAccount={this.props.reimbursementAccount}
+                <Form
+                    formID={ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM}
+                    validate={this.validate}
                     onSubmit={this.submit}
+                    submitButtonText={this.props.translate('common.saveAndContinue')}
+                    style={[styles.mh5, styles.flexGrow1]}
                 >
                     <Text style={[styles.mb5]}>
                         <Text>{this.props.translate('beneficialOwnersStep.checkAllThatApply')}</Text>
                     </Text>
                     <CheckboxWithLabel
+                        inputID="ownsMoreThan25Percent"
                         style={[styles.mb2]}
-                        isChecked={this.state.ownsMoreThan25Percent}
-                        onInputChange={() => this.toggleCheckbox('ownsMoreThan25Percent')}
                         LabelComponent={() => (
                             <Text>
                                 {this.props.translate('beneficialOwnersStep.iOwnMoreThan25Percent')}
                                 <Text style={[styles.textStrong]}>{this.props.companyName}</Text>
                             </Text>
                         )}
+                        onValueChange={(value) => {
+                            this.setState({ownsMoreThan25Percent: value});
+                            if (value && this.state.beneficialOwners.length > 3) {
+                                // if ownsMoreThan25Percent is true, then there can only be 3 IdentityForms so remove the last one
```

Comment:
NAB but I think this is a bit more descriptive

```suggestion
                                // If the user owns more than 25% of the company, then there can only be a maximum of 3 other beneficial owners who owns more than 25%. We have to remove the 4th beneficial owner if the checkbox is checked.
```